logstash-core/src/main/java/org/logstash/health/Probe.java

```diff
@@ -45,7 +45,7 @@ public static class Builder {
             private final Impact impact;
 
             public Builder() {
-                this(Status.UNKNOWN, null, null);
+                this(Status.GREEN, null, null);
```

Comment:
Did status order change?

Reply:
All existing usages also send `Builder#withStatus`, so this is a no-op from a usage case.

Since we made `GREEN` be less-degraded than `UNKNOWN`, this should default to the least-degraded state as was the initial intent.